<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="output">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>491</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Tahoma&quot;;
background-color: white;
border-radius: 15px;
</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="inputB">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>65</y>
      <width>491</width>
      <height>55</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
border-radius: 15px;
font: 10pt &quot;Tahoma&quot;;
padding: 2px;</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Tahoma'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openFile">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>65</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/open.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Tahoma&quot;;</string>
    </property>
    <property name="text">
     <string>LR3|Bolts and nuts</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="inputN">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>491</width>
      <height>55</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
border-radius: 15px;
font: 10pt &quot;Tahoma&quot;;
padding: 2px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameN">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Tahoma&quot;;</string>
    </property>
    <property name="text">
     <string>Nuts array:</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameB">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>35</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Tahoma&quot;;</string>
    </property>
    <property name="text">
     <string>Bolts array:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkStep">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Tahoma&quot;;</string>
    </property>
    <property name="text">
     <string>Step by step</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>180</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ok.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkInfo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Tahoma&quot;;</string>
    </property>
    <property name="text">
     <string>Full info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveFile">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>365</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/savef.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
